mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -23,15 +23,23 @@ import org.apache.spark.Logging
 import org.apache.spark.mllib.linalg.Vectors
 import org.apache.spark.mllib.stat.{MultivariateStatisticalSummary, MultivariateOnlineSummarizer}
 import org.apache.spark.sql.DataFrame
-
 /**
  * Evaluator for regression.
  *
- * @param predictionAndObservations an RDD of (prediction, observation) pairs.
+ * @param predictionAndObservations an RDD of (prediction, observation) pairs,
+ * @param regThroughOrigin true if intercept is not included in linear regression model
```

Comment:
That makes sense. Could you change `regThroughOrigin` to `throughOrigin` since reg is often used as regularization. Also, could you update the scala doc saying that `True if the regression is through the origin. For example, in linear regression, it will be true without fitting intercept`.